### ISSUE TITLE
New 2.0.6 Errors

### ISSUE THREAD

**Issue by blessedbro** [Motyldrogi/fansly-downloader#18]
This new version was supposed to fix an issue with downloading videos, but it seems to be broken. It does not do anything when clicking the download buttons. I also see some errors in the chrome extensions area. Images do download without any issues for me.

<img width="356" alt="Screenshot 2023-05-04 215634" src="https://user-images.githubusercontent.com/105473987/236368913-1092e0ea-cfd2-4fae-92e1-72f2bd1421d7.png">

**Comment by Motyldrogi** [comment]
@blessedbro Before you made the update, did the video download work?

Also can you pls check your version of fansly (Click on profile icon top right -> Settings -> About)

I have 3.0.2 Version of Fansly where the video download was broken with the 2.0.5 Downloader, so the 2.0.6 Downloader fixes this.
I guess that maybe everything under 3.0.2 Fansly only works with the 2.0.5 Downloader?

**Comment by NMPopsicle** [comment]
+1 for experiencing issues on 2.0.6, but only on certain videos. Videos where the quality gear offers "Auto HD" option, do not download. Videos that do not offer this option work just fine with 2.0.6 - Both Fanly and the downloader are up-to-date

![image](https://user-images.githubusercontent.com/104788767/236374175-4d4a352e-084d-4a55-829c-bf8779cda7d9.png)

**Comment by blessedbro** [comment]
@Motyldrogi Yes, I am on 3.0.2 Fansly. 2.0.5 does not download the videos at all. 2.0.6 does work with some videos. Just as @NMPopsicle pointed out. the auto hd videos appear to be the issue, and from what I can tell, they are all newer uploads. with in the past day or two. All of the older videos that do not have the auto hd seem to work fine with the 2.0.6 version.

**Comment by noonotthat** [comment]
@Motyldrogi and  @NMPopsicle  I can confirm I am having the same issue with videos that have the "auto" under quality settings.

**Comment by PlzHlpWithFansly** [comment]
Same issue here...

Uncaught (in promise) TypeError: Failed to fetch

points to line 75 "   fetch(downloadLink)"

**Comment by Hot-Hot-Fire** [comment]
I'm having the same issue.

**Comment by alfratrople** [comment]
Same issue here.

**Comment by AlexMariacio** [comment]
Same with me, videos with "auto" resolution can't be downloaded.

**Comment by WeebSquatch** [comment]
I can't download any videos either, even when selecting resolution

**Comment by KostiG7** [comment]
download button doesn't work!
![f](https://github.com/Motyldrogi/fansly-downloader/assets/146674322/e804a93b-b4ed-48af-aec0-a019b9449987)

**Comment by baszero** [comment]
failed to fetch. same issue

**Comment by Eurania343** [comment]
I've got the same issue. Still not fixed? Dang, was hoping for a workaround.

Uncaught TypeError: Cannot read properties of undefined (reading 'split') at line 182
182 - dconst name = feedUsername + "-" + downloadLink.split("/")[4].split("?")[0];

I also get the Failed to Fetch at line 75.

74 - if (downloadLink != null && !downloadLink.includes("mp4")) {
75 -     fetch(downloadLink)
